.github/workflows/ci.yml

```diff
@@ -87,7 +87,12 @@ jobs:
         run: |
           docker push "$CORE_CI_IMAGE:latest"
           docker push "$CORE_CI_IMAGE:ci--$BRANCH_REF"
-      - name: Run Rubocop linting
+      - name: Run Python flake8 linting
+        if: matrix.suite == 'python'
+        run: |
+          sudo apt-get install python3-pip
```

Comment:
Do we need to install python3 on the host image if we're running pip inside the container?

The container should already have `pip` installed via `pyenv` and it's available as `pyenv exec pip`, but I suggest we add flake8 to the requirements.txt so it gets cached in the docker build, and then we can run `pyenv exec flake8` here.

FYI if you want to test this out locally, the easiest way is to clone the project and run `bin/docker-dev-shell`, it will give you the actual container that it used in the `docker run` step below